packages/browser/src/ClientAuthentication.ts

```diff
@@ -84,14 +94,14 @@ export default class ClientAuthentication {
   };
 
   // By default, our fetch() resolves to the environment fetch() function.
-  fetch = window.fetch;
+  fetch = fetchWithCookies;
 
   logout = async (sessionId: string): Promise<void> => {
     await this.logoutHandler.handle(sessionId);
 
     // Restore our fetch() function back to the environment fetch(), effectively
     // leaving us with un-authenticated fetches from now on.
-    this.fetch = window.fetch;
+    this.fetch = fetchWithCookies;
```

Comment:
Both very good points: 81281258f3ac7c28fe4834eb6e7f6045b04028b0 and 4c4455ea084ef31f86a04ed991a5119ae2db8d68

Reply:
:+1: Any reason it couldn't be set to `window.fetch` like before?

Reply:
If set to `windows.fetch`, fetching after logging out threw the error `Calling 'fetch' on an object not implementing the Window interface`, which was also an issue documented in https://github.com/inrupt/solid-client-authn-js/pull/921/commits/520a19bdf9ca2d4e6e7e377a3dcf24053cf4ba70#diff-4f6320deb9e43b5216428fb7c2dd230f4b0c5325c6f147f6a73b15659c355fb9L132

Reply:
I think that should be solvable by setting it to `window.fetch.bind(window)`, or possibly by just referring to the `fetch` global, rather than wrapping it in a wrapper function with identical signature. Not a biggie, but would be future-proof against updates of the Fetch API.